MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -1182,6 +1202,19 @@
 			name = Resources;
 			sourceTree = "<group>";
 		};
+		3529FCEE21A5C52C00AEA9AA /* Recovered References */ = {
```

Comment:
Check whether these recovered references belong anywhere else in the project; if not, feel free to delete the group.